src/dbgshim/debugshim.cpp

```diff
@@ -611,18 +726,36 @@ HRESULT CLRDebuggingImpl::GetCLRInfo(ICorDebugDataTarget* pDataTarget,
     else
 #endif // !HOST_WINDOWS
     {
-        swprintf_s(pDacName, dwDacNameCharCount, W("%s"), MAKEDLLNAME_W(CORECLR_DAC_MODULE_NAME_W));
-        swprintf_s(pDbiName, dwDbiNameCharCount, W("%s"), MAKEDLLNAME_W(MAIN_DBI_MODULE_NAME_W));
+        clrInfo.WindowsTarget = FALSE;
 
-        pVersion->wMajor = 0;
-        pVersion->wMinor = 0;
-        pVersion->wBuild = 0;
-        pVersion->wRevision = 0;
+        uint64_t symbolAddress;
+        if (TryGetSymbol(pDataTarget, moduleBaseAddress, RUNTIME_INFO_SIGNATURE, &symbolAddress))
+        {
+            RuntimeInfo runtimeInfo;
+            ULONG32 bytesRead;
+            if (SUCCEEDED(pDataTarget->ReadVirtual(symbolAddress, (BYTE*)&runtimeInfo, sizeof(RuntimeInfo), &bytesRead)))
+            {
+                if (strcmp(runtimeInfo.Signature, RUNTIME_INFO_SIGNATURE) == 0)
+                {
+                    clrInfo.IndexType = LIBRARY_PROVIDER_INDEX_TYPE::Identity; 
 
-        *pdwDbiTimeStamp = 0;
-        *pdwDbiSizeOfImage = 0;
-        *pdwDacTimeStamp = 0;
-        *pdwDacSizeOfImage = 0;
+                    // The first byte is the number of bytes in the index
+                    clrInfo.DbiBuildIdSize = runtimeInfo.DbiModuleIndex[0];
```

Comment:
This branch only applies to single file, right? Do you mind if we add a comment where we expect usual scenarios to go?

Reply:
Done.